sdks/uniswapx-sdk/src/builder/V2DutchOrderBuilder.ts

```diff
@@ -287,9 +287,9 @@ export class V2DutchOrderBuilder extends OrderBuilder {
       "exclusivityOverrideBps not set"
     );
     invariant(
-      this.info.cosignerData.inputOverride !== undefined &&
+      this.info.cosignerData.inputOverride !== undefined && // inputOverride is defaulted to 0 because enforced to be of type BigNumber
```

Comment:
nit: can you move the comment to its own line instead of inline?

If inputOverride is defaulted to `0`, why do we check if it's not `undefined`?

Reply:
This is a `V2` file where I spotted this mistake and was just making a note of it since I noticed this condition doesn't do anything. I will go ahead and remove it.